tests/test_idpi/test_time_ops.py

```diff
@@ -43,3 +43,32 @@ def test_delta(data_dir, fieldextra):
     expected = xr.concat([fx_ds["tot_prec_03h"] for fx_ds in fx_ds_h], dim="time")
 
     assert_allclose(observed, expected.transpose("epsd_1", "time", ...))
+
+
+def test_max(data_dir, fieldextra):
```

Comment:
It would be nice some of the raises, by constracting some artificial arrays with non uniform time coordinates or non multiple steps.